Source/Charts/Renderers/XAxisRenderer.swift

```diff
@@ -173,15 +173,15 @@ open class XAxisRenderer: AxisRendererBase
             else { return }
         
         #if os(OSX)
-            let paraStyle = NSParagraphStyle.default.mutableCopy() as! NSMutableParagraphStyle
+        let paraStyle = NSParagraphStyle.default.mutableCopy() as! NSMutableParagraphStyle
```

Comment:
seems like `import UIKit` within `#if` has indent but here it does not. We need to unify

Reply:
I will actually remove this one since it's the same API on macOS and iOS. In fact, `OSX` isn't even valid anymore.